### PR TITLE
Update CHANGELOG and VERSION for v0.3.0 release

### PR DESCRIPTION
**Pull request by ziyouw**
Set VERSION to v0.3.0
Create CHANGELOG file for 0.3.0 release.
List new features and issues fixed

Signed-off-by: Ziyou Wang <ziyouw@vmware.com>

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v0.3.0-dev
+v0.3.0
```
Comment (yanjunz97) [inline VERSION:1]: We may want to change VERSION to v0.4.0-dev when merging to main while v0.3.0 when merging to release-0.3?

Comment (ziyouw) [reply VERSION:1]: Yes, the purpose is we need to change it to v0.3.0 for release-0.3 branch. so either we change it in main now or we change it in the new branch.

File: CHANGELOG/CHANGELOG-0.3.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Changelog 0.3
+
+## 0.3.0 - 2022-10-19
+
+### Added
+
+- Theia Manager. ([#97](https://github.com/antrea-io/theia/pull/97), [@wsquan171]) ([#109](https://github.com/antrea-io/theia/pull/109), [@wsquan171]) ([#111](https://github.com/antrea-io/theia/pull/111), [@yuntanghsu]) ([#113](https://github.com/antrea-io/theia/pull/113), [@wsquan171]) ([#114](https://github.com/antrea-io/theia/pull/114), [@yanjunz97]) ([#115](https://github.com/antrea-io/theia/pull/115), [@yuntanghsu]) 
+  * Introduce a new component Theia Manager to implement the control logics
+  * Update policy recommendation and CLIs to use Theia Manager
```
Comment (salv-orlando) [inline CHANGELOG/CHANGELOG-0.3.md:9]: Why did you chose plural "CLIs"? I think we only have one :)

File: CHANGELOG/CHANGELOG-0.3.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Changelog 0.3
+
+## 0.3.0 - 2022-10-19
+
+### Added
+
+- Theia Manager. ([#97](https://github.com/antrea-io/theia/pull/97), [@wsquan171]) ([#109](https://github.com/antrea-io/theia/pull/109), [@wsquan171]) ([#111](https://github.com/antrea-io/theia/pull/111), [@yuntanghsu]) ([#113](https://github.com/antrea-io/theia/pull/113), [@wsquan171]) ([#114](https://github.com/antrea-io/theia/pull/114), [@yanjunz97]) ([#115](https://github.com/antrea-io/theia/pull/115), [@yuntanghsu]) 
+  * Introduce a new component Theia Manager to implement the control logics
+  * Update policy recommendation and CLIs to use Theia Manager 
+- Enhanced UI. ([#101](https://github.com/antrea-io/theia/pull/101), [@heanlan]) ([#102](https://github.com/antrea-io/theia/pull/102), [@yuntanghsu])
```
Comment (salv-orlando) [inline CHANGELOG/CHANGELOG-0.3.md:10]: nit: Consider "Grafana UI enhancements"

File: CHANGELOG/CHANGELOG-0.3.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  * Introduce a Grafana home dashboard
+  * Improvement for throughput related panels in Grafana
+- Support for Snowflake. ([#112](https://github.com/antrea-io/theia/pull/112), [@antoninbas]) ([#118](https://github.com/antrea-io/theia/pull/118), [@heanlan]) ([#119](https://github.com/antrea-io/theia/pull/119), [@heanlan])
+  * Introduce Snowflake backend for Theia
```
Comment (salv-orlando) [inline CHANGELOG/CHANGELOG-0.3.md:14]: nit: add support for Snowflake as a database backend for Theia.

File: CHANGELOG/CHANGELOG-0.3.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  * Enhance UI support for Snowflake
+
+### Changed
+
+- Adopt Golang version ClickHouse data schema management ([#95](https://github.com/antrea-io/theia/pull/95), [@yanjunz97])
```
Comment (salv-orlando) [inline CHANGELOG/CHANGELOG-0.3.md:19]: please change this to something like:

`Adopt golang-migrate` for ClickHouse data schema management

**Review by salv-orlando — COMMENTED** [review]
Some inline comments.
I will shortly create the release branch, we should retarget this PR to the release branch (it will conflict, the 0.2 changelog will need to be removed)

**Comment by salv-orlando** [comment]
Changed base to release-0.3.
Thanks @ziyouw for addressing the comments.

**Pull request merged**